packages/ckeditor5-ui/tests/colorpicker/colorpickerview.js

```diff
@@ -269,4 +417,18 @@ describe( 'ColorPickerView', () => {
 			sinon.assert.calledOnce( spy );
 		} );
 	} );
+
+	function testColorUpdateFromInput( options ) {
+		const fieldView = view.inputRow.children.get( 1 ).fieldView;
+		view.color = '#000000';
+
+		fieldView.isFocused = true;
+		fieldView.value = options.inputValue;
+		fieldView.fire( 'input' );
+
+		clock.tick( 200 );
+
+		expect( fieldView.value, 'Wrong input value' ).to.equal( options.expectedInput );
```

Comment:
Thanks for not forgetting to properly name similar assertions. Something that is notoriously skipped in other parts of our code base.